### PR TITLE
support for multiple regions

### PR DESCRIPTION
**Pull request by goya**
Supports:

- `discovery.job[].regions`
- `static[].regions`

to allow specifying multiple regions.  

This new property overrides `region` only if `region` is not specified.

### CODE REVIEW

**Comment by thomaspeitz** [comment]
Let us kill region and make it the new default.

What is your opinion about it?

**Comment by goya** [comment]
added the readme update.

File: config.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ type exportedTagsOnMetrics map[string][]string
 
 type job struct {
 	Region        string   `yaml:"region"`
```
Comment (thomaspeitz) [inline config.go:24]: There seems to be still region in there. Can we remove this?

File: abstract.go [hunk 2]
```diff
@@ -16,6 +16,16 @@ var (
 	tagSemaphore        chan struct{}
 )
 
+func getRegions(region string, regions []string) []string {
```
Comment (thomaspeitz) [inline abstract.go:19]: Whole logic is no more needed as far as i can see, if we remove region.

File: config.go (hunk 1 above)
Comment (goya) [reply config.go:24]: i'll remove it.  kept it in so it wouldnt be a breaking change, just new functionality - at least for a few versions.

File: abstract.go (hunk 2 above)
Comment (goya) [reply abstract.go:19]: same as above, i'll update it.

**Comment by thomaspeitz** [comment]
Thanks a lot :) Will release the following days!

**Comment by goya** [comment]
this closes issue #160

**Pull request merged**